From: Jochen Sprickerhof <jspricke@debian.org>
Date: Thu, 22 Aug 2024 12:21:43 +0200
Subject: Add missing include to build with gcc-14

Closes: #1074815
---
 src/cmdline/cmdline_show.cc | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/cmdline/cmdline_show.cc b/src/cmdline/cmdline_show.cc
index 8277756..d1bee53 100644
--- a/src/cmdline/cmdline_show.cc
+++ b/src/cmdline/cmdline_show.cc
@@ -52,6 +52,7 @@
 #include <apt-pkg/version.h>
 
 #include <iostream>
+#include <algorithm>
 
 namespace cw = cwidget;
 using aptitude::cmdline::create_terminal;
